### PR TITLE
issue #1660 : fix broken markdown tags

### PR DESCRIPTION
**Pull request by cbeauchesne**
hop

Related to #1660

### CODE REVIEW

File: c2corg_ui/format/__init__.py [hunk 1]
```diff
@@ -62,5 +63,11 @@ def parse_code(text, md=True, bb=True):
...
 
+_sanitize_fixer = re.compile(r"&lt;(/?)(sup|sub|br|p|abbr|del|ins)&gt;")
+
+
 def sanitize(text):
-    return html.escape(text)
+    escaped = html.escape(text)
+    blockquote_fixed = escaped.replace("\n&gt;", "\n>")
+    html_fixed = _sanitize_fixer.sub(r"<\1\2>", blockquote_fixed)
+    return html_fixed
```
Comment (asaunier) [inline c2corg_ui/format/__init__.py]: Very minor but you don't really need this intermediate variable here, you could directly write
```
return _sanitize_fixer.sub(r"<\1\2>", blockquote_fixed)
```

**Comment by asaunier** [comment]
Thanks @cbeauchesne that looks good to me.

Just one tip when creating PRs: it is useful to add a link to the related issues in the initial comment (it automatically creates backlinks in the issues to the PR). You can also label the issues with the "pull request ready" tag which are then turned to "ready for demo" when the PR is merged.

File: c2corg_ui/format/__init__.py (hunk 1 above)
Comment (cbeauchesne) [reply c2corg_ui/format/__init__.py]: yeah, I wanted to be crystal clear :)

**Comment by cbeauchesne** [comment]
mmmh, I think I can't add a label on an issue, did I miss something?

**Comment by asaunier** [comment]
@cbeauchesne Feel free to merge this PR.

**Comment by cbeauchesne** [comment]
It will be more accurate to work on onlymarkdown directly

**Pull request closed without merging**